contracts/axelar-interchain-token-service/src/error.rs

```diff
@@ -8,5 +8,5 @@ pub enum ContractError {
     AlreadyInitialized = 2,
```

Comment:
remove explicit numbering from the error

Reply:
It will not compile without explicit numbering: `enums are supported as contract errors only when all variants have an explicit integer literal`